ssl/ssl_ciph.c

```diff
@@ -2148,6 +2148,16 @@ int SSL_CIPHER_get_auth_nid(const SSL_CIPHER *c)
     return ssl_cipher_table_auth[i].nid;
 }
 
+int ssl_get_md_idx(int md_nid) {
+    int i;
+
+    for(i=0; i<SSL_MD_NUM_IDX; i++) {
```

Comment:
Nit: spaces around `=` and `<`

Reply:
So changed in 0ac814f8247d1a5f157eec2b62bce0000c3a2d1e.